CHARTER.md

```diff
@@ -0,0 +1,63 @@
+
+## Section 1: Scope
+
+`vis.gl` is to provide a comprehensive, open-source solution for GPU-powered visualizations of large-scale datasets, in particular geospatial data visualizations. The `vis.gl` project consists of many different sub-projects, such as a layer-based visualization framework, GPU toolkit, basemap react components, and more. The project is designed to be modular, so that it can be used in a variety of different environments and for different purposes.
+
+The scope of project includes documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open source project.
+
+### 1.1: In-scope
+
+ - Overseeing the **technical development** of the core vis.gl libraries and all affiliated sub-projects.
+ - **Release planning** and coordination
+ - Managing project repositories and **governance**, including issue tracking, pull request processes, and repository organization.
+ - Defining and enforcing **coding standards, testing practices, and development processes** to maintain code quality.
+ - Maintaining the list of **Maintainers and core contributors**, and setting permissions in project repositories.
+ - Reviewing and proposing **new project incubations or sub-projects**, as well as deciding on deprecations or significant changes to the project’s scope.
+ - Formation of technical **working groups**.
+ - **Representation** in technical collaborations with external communities.
+
+### 1.2: Out-of-Scope
+
+Section Intentionally Left Blank
+
+## Section 2: Relationship with OpenJS Foundation CPC.
+
+Technical leadership for the projects within the [OpenJS Foundation](https://openjsf.org/) is delegated to the projects through their project charters by the [OpenJS Foundation Cross-Project Council](https://openjsf.org/about/governance/) (CPC). In the case of the `vis.gl` project, it is delegated to the [Steering Committee Members](https://github.com/visgl/tsc/blob/master/README.md#steering-committee-members) (the “TSC”). The OpenJS Foundation's business leadership is the Board of Directors (the “Board”).
+
+This `vis.gl` Charter reflects a carefully constructed balanced role for the TSC and the CPC in the governance of the OpenJS Foundation. The charter amendment process is for the TSC to propose changes using simple majority of the full TSC, the proposed changes being subject to review and approval by the CPC. The CPC may additionally make amendments to the project charter at any time, though the CPC will not interfere with day-to-day discussions, votes, or meetings of the TSC.
+
+### 2.1 Other Formal Project Relationships
+
+Section Intentionally Left Blank
+
+## Section 3: `vis.gl`'s TSC Governing Body
+
+`vis.gl` is governed by its [Steering Committee Members](https://github.com/visgl/tsc/blob/master/README.md#steering-committee-members).
```

Comment:
Either use TSC or SCM, but don't do both.

Also, it is the governing body that has decision making power, not its members.

Reply:
Ok, this is something I was unsure of. Other charters like NVM use "Maintainers", the role, instead of a TSC, a body.

I'm on board with using TSC in the charter. What is the way a TSC usually differentiates the people who are providing leadership and voting responsibility vs the people who are only delegated some responsibilities? We current define the leadership group "Steering committee members" and the other group "Maintainers" - is this fine or is it unclear as-written?

Reply:
I'm just suggesting to go for consistency here. The charter uses "Steering Committee", "TSC", and "Steering Committee Members." Just pick one and go with it. I'd advise using TSC as that's what others do.